include/model/statedata.h

```diff
@@ -73,11 +73,10 @@ NODE_STATE_DECLARE(sched_context_t, *ksIdleSC);
 #endif
 
 #ifdef CONFIG_HAVE_FPU
-/* Current state installed in the FPU, or NULL if the FPU is currently invalid */
-NODE_STATE_DECLARE(user_fpu_state_t *, ksActiveFPUState);
-/* Number of times we have restored a user context with an active FPU without switching it */
-NODE_STATE_DECLARE(word_t, ksFPURestoresSinceSwitch);
+/* Current thread using the FPU, or NULL if FPU state is invalid */
```

Comment:
suggestion: `The thread currently using the FPU, ...` (I was momentarily confused that this indicated the current thread is using the FPU)

Reply:
Good point, but any use of "current" may be confusing. What about `The thread currently using the FPU, ...`?

Reply:
That phrase happens to be true, but is that what you want to communicate to people? There's an association/ownership aspect here that might be important.
Otherwise sure, that will work.

Reply:
I'm trying to make it clearer what the variable means. I could use the more verbose:

`Thread whose current state is installed in the FPU, or NULL if the FPU is currently invalid`

Reply:
As a sort of data point for this, in the specifications I'm currently calling the equivalent variable `arm_current_fpu_owner`. Do you think a name like that makes sense and could be used here, or should I be avoiding 'current' there as well?

Reply:
For consistency with the naming of other current states, it should then be `ksCurFPUOwner`. I'm fine with changing it, I'm not attached to the `ksActiveFPUThread` naming.